client/app/hosting/multisite/add/external/hosting-multisite-add-external.html

```diff
@@ -57,6 +57,26 @@
         </div>
     </div>
 
+    <div class="form-group" data-ng-if="hosting.isCloudWeb">
+        <label class="control-label col-md-4" for="selectedRuntime" data-i18n-static="hosting_tab_DOMAINS_configuration_modify_step1_runtime"></label>
+        <div data-ng-if="loaders.runtimes">
+            <oui-spinner data-size="s"></oui-spinner>
+        </div>
+
+        <div data-ng-if="!loaders.runtimes">
```

Comment:
You can apply `data-ng-if="!loaders.runtimes"` directly to `<div class="col-md-8">`.